components/menu/index.en-US.md

```diff
@@ -129,17 +129,17 @@ The legacy demo code for version `<4.20.0` could be found at [https://github.com
 
 #### SubMenuType
 
-| Param | Description | Type | Default value | Version |
-| --- | --- | --- | --- | --- | --- |
-| children | Sub-menus or sub-menu items | [ItemType\[\]](#ItemType) | - |  |
-| disabled | Whether sub-menu is disabled | boolean | false |  |
-| icon | Icon of sub menu | ReactNode | - |  |
-| key | Unique ID of the sub-menu | string | - |  |
-| label | Menu label | ReactNode | - |  |
-| popupClassName | Sub-menu class name, not working when `mode="inline"` | string | - |  |
-| popupOffset | Sub-menu offset, not working when `mode="inline"` | \[number, number] | - |  |
-| theme | Color theme of the SubMenu (inherits from Menu by default) |  | `light` \| `dark` | - |  |
-| onTitleClick | Callback executed when the sub-menu title is clicked | function({ key, domEvent }) | - |  |
+| Param          | Description                                                | Type                        | Default value | Version |
+| ---------------| ---------------------------------------------------------- | --------------------------- | ------------- | ------- |
+| children       | Sub-menus or sub-menu items                                | [ItemType\[\]](#ItemType)   | -             |         |
+| disabled       | Whether sub-menu is disabled                               | boolean                     | false         |         |
+| icon           | Icon of sub menu                                           | ReactNode                   | -             |         |
+| key            | Unique ID of the sub-menu                                  | string                      | -             |         |
+| label          | Menu label                                                 | ReactNode                   | -             |         |
+| popupClassName | Sub-menu class name, not working when `mode="inline"`      | string                      | -             |         |
+| popupOffset    | Sub-menu offset, not working when `mode="inline"`          | \[number, number]           | -             |         |
+| theme          | Color theme of the SubMenu (inherits from Menu by default) | `light` \| `dark`           | -             |         |
+| onTitleClick   | Callback executed when the sub-menu title is clicked       | function({ key, domEvent }) | -             |         |
```

Comment:
| Param | Description | Type | Default value | Version |
| --- | --- | --- | --- | --- |
| children | Sub-menus or sub-menu items | [ItemType\[\]](#ItemType) | - |  |
| disabled | Whether sub-menu is disabled | boolean | false |  |
| icon | Icon of sub menu | ReactNode | - |  |
| key | Unique ID of the sub-menu | string | - |  |
| label | Menu label | ReactNode | - |  |
| popupClassName | Sub-menu class name, not working when `mode="inline"` | string | - |  |
| popupOffset | Sub-menu offset, not working when `mode="inline"` | \[number, number] | - |  |
| theme | Color theme of the SubMenu (inherits from Menu by default) |  | `light` \| `dark` | - |  |
| onTitleClick | Callback executed when the sub-menu title is clicked | function({ key, domEvent }) | - |  |